src/components/Main/Results/ResultsCard.tsx

```diff
@@ -74,16 +78,14 @@ function ResultsCard({ canRun, severity, result, caseCounts }: ResutsCardProps)
   return (
     <CollapsibleCard
       identifier="results-card"
-      title={<h3 className="p-0 m-0 text-truncate">Results</h3>}
-      help="This section contains simulation results"
+      title={<h3 className="p-0 m-0 text-truncate">{t('Results')}</h3>}
+      help={t('This section contains simulation results')}
       defaultCollapsed={false}
     >
       <Row noGutters>
         <Col>
           <p>
-            {`This output of a mathematical model depends on model assumptions and parameter choices.
-              We have done our best (in limited time) to check the model implementation is correct.
-              Please carefully consider the parameters you choose and interpret the output with caution.`}
+            {t('Results disclaimer')}
```

Comment:
Am not sure you really want to substitute the text to translate with a generic label